### PR TITLE
feature(python-logic) - download file directly

### PR DESCRIPTION
**Pull request by giladlevy7**
We changed the API to redirect the request to download the file directly and avoid an extra request

### CODE REVIEW

File: rivery_cli/converters/entities/rivers.py [hunk 1]
```diff
@@ -300,7 +300,9 @@ def step_importer(cls, steps: list, rivery_session: RiverySession, code_dir: str
                 current_step["loop_over_value"] = step.pop("loop_over_value", "")
                 current_step["loop_over_variable_name"] = step.pop("loop_over_variable_name", [])
                 current_step["steps"] = cls.step_importer(
-                    steps=step.pop('nodes', [])
+                    steps=step.pop('nodes', []),
+                    rivery_session=rivery_session,
+                    code_dir=code_dir
```
Comment (Alonreznik) [inline rivery_cli/converters/entities/rivers.py:305]: why there's a need on rivery_session there?

Comment (giladlevy7) [reply rivery_cli/converters/entities/rivers.py]: @Alonreznik  it was this way in the initial commit, just changed it, please take another look

File: rivery_cli/converters/entities/rivers.py [hunk 2]
```diff
@@ -362,4 +371,4 @@ def _import(cls, def_: dict, rivery_session: RiverySession, code_dir: str) -> di
 
         final_response[global_keys.BASE][global_keys.DEFINITION] = definition_
 
-        return final_response
+        return final_response, files_to_download
```
Comment (Alonreznik) [inline rivery_cli/converters/entities/rivers.py:374]: It's better to add the files to download inside the final response

File: rivery_cli/utils/logicode_utils.py [hunk 3]
```diff
@@ -1,34 +1,16 @@
...
 import os
 
-import requests
 
-from rivery_cli.rivery_session import RiverySession
-
-
-def download_python_file(file_id: str, rivery_session: RiverySession, code_dir: str, file_name: str):
-    # Get the actual path from project.yaml
+def get_python_file_to_download(file_id: str, code_dir: str, file_name: str):
```
Comment (Alonreznik) [inline rivery_cli/utils/logicode_utils.py:4]: I would change the name from pytjon to just "get file to download"

**Review by Alonreznik — APPROVED** [review]

**Pull request merged**